### ISSUE TITLE
whitespace rule: false positive for handling () as a function call

### ISSUE THREAD

**Issue by robert-zaremba** [duaraghav8/Solium#104]
The following code represents false-positive `whitespace` warning:
```
require(msg.sender == address(wallet.directory().brg()));
                                                   ^-- WARNING: 'address': The first argument must not be preceded by any whitespace or comments (only '(').
```

**Comment by mcdee** [comment]
Same error in a slightly different situation:
```
            balance = balance.add(balance.mul(dividends[currentDividend].amount).div(dividends[currentDividend].supply));
                                                                                   ^-- WARNING: 'undefined': The first argument must not be preceded by any whitespace or comments (only '(').                                                                          
```

(.add, .mul, .div are from safemath).

**Comment by oed** [comment]
I have the same error when using require:
```
require(destination.call.value(value)(data));
                                    ^-- WARNING: 'require': The first argument must not be preceded by any whitespace or comments (only '(').
```

**Comment by duaraghav8** [comment]
Fixed completely in `v1.0.4`, thanks for reporting guys!
